2024-03-09 10:52:23,147 INFO    MainThread:408 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-03-09 10:52:23,148 INFO    MainThread:408 [wandb_setup.py:_flush():76] Configure stats pid to 408
2024-03-09 10:52:23,148 INFO    MainThread:408 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\.config\wandb\settings
2024-03-09 10:52:23,148 INFO    MainThread:408 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\Desktop\CV\Assignment_2\wandb\settings
2024-03-09 10:52:23,148 INFO    MainThread:408 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-09 10:52:23,148 INFO    MainThread:408 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-09 10:52:23,161 INFO    MainThread:408 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-09 10:52:23,161 INFO    MainThread:408 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240309_105223-qi8utcax\logs\debug.log
2024-03-09 10:52:23,161 INFO    MainThread:408 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240309_105223-qi8utcax\logs\debug-internal.log
2024-03-09 10:52:23,162 INFO    MainThread:408 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x000001EB32AE8BD0>
2024-03-09 10:52:23,162 INFO    MainThread:408 [wandb_init.py:init():566] calling init triggers
2024-03-09 10:52:23,162 INFO    MainThread:408 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'regularization_parameter': 0.001, 'max_iter': 1000}
2024-03-09 10:52:23,163 INFO    MainThread:408 [wandb_init.py:init():616] starting backend
2024-03-09 10:52:23,163 INFO    MainThread:408 [wandb_init.py:init():620] setting up manager
2024-03-09 10:52:23,164 INFO    MainThread:408 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-09 10:52:23,167 INFO    MainThread:408 [wandb_init.py:init():628] backend started and connected
2024-03-09 10:52:23,174 INFO    MainThread:408 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-09 10:52:23,182 INFO    MainThread:408 [wandb_init.py:init():720] updated telemetry
2024-03-09 10:52:23,189 INFO    MainThread:408 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-09 10:52:24,313 INFO    MainThread:408 [wandb_run.py:_on_init():2262] communicating current version
2024-03-09 10:52:24,645 INFO    MainThread:408 [wandb_run.py:_on_init():2271] got version response 
2024-03-09 10:52:24,646 INFO    MainThread:408 [wandb_init.py:init():804] starting run threads in backend
2024-03-09 10:52:36,148 INFO    MainThread:408 [wandb_run.py:_console_start():2241] atexit reg
2024-03-09 10:52:36,148 INFO    MainThread:408 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-09 10:52:36,149 INFO    MainThread:408 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-09 10:52:36,149 INFO    MainThread:408 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-09 10:52:36,149 INFO    MainThread:408 [wandb_init.py:init():847] run started, returning control to user process
2024-03-09 11:10:23,804 INFO    MainThread:408 [wandb_setup.py:_flush():76] Current SDK version is 0.16.4
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_setup.py:_flush():76] Configure stats pid to 408
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\.config\wandb\settings
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_setup.py:_flush():76] Loading settings from C:\Users\hp\Desktop\CV\Assignment_2\wandb\settings
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_init.py:_log_setup():526] Logging user logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240309_111023-1tkaifwf\logs\debug.log
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_init.py:_log_setup():527] Logging internal logs to C:\Users\hp\Desktop\CV\Assignment_2\wandb\run-20240309_111023-1tkaifwf\logs\debug-internal.log
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_init.py:init():566] calling init triggers
2024-03-09 11:10:23,805 INFO    MainThread:408 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'regularization_parameter': 0.001, 'max_iter': 2000}
2024-03-09 11:10:23,806 INFO    MainThread:408 [wandb_init.py:init():591] re-initializing run, found existing run on stack: qi8utcax
2024-03-09 11:10:23,807 INFO    MainThread:408 [wandb_run.py:_finish():1970] finishing run tejas591995/SIFT-SVM-MoVW-3/qi8utcax
2024-03-09 11:10:23,807 INFO    MainThread:408 [jupyter.py:save_history():445] not saving jupyter history
2024-03-09 11:10:23,807 INFO    MainThread:408 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-09 11:10:23,807 INFO    MainThread:408 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-09 11:10:23,807 INFO    MainThread:408 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-09 11:10:23,807 INFO    MainThread:408 [wandb_run.py:_restore():2193] restore
2024-03-09 11:10:23,808 INFO    MainThread:408 [wandb_run.py:_restore():2199] restore done
2024-03-09 11:10:28,757 INFO    MainThread:408 [wandb_run.py:_footer_history_summary_info():3851] rendering history
2024-03-09 11:10:28,757 INFO    MainThread:408 [wandb_run.py:_footer_history_summary_info():3883] rendering summary
2024-03-09 11:10:28,769 INFO    MainThread:408 [wandb_run.py:_footer_sync_info():3810] logging synced files
